src/util/Constructor.re

```diff
@@ -0,0 +1,7 @@
+//open OptUtil.Syntax;
+open Sexplib.Std;
```

Comment:
not sure this should be in util since its related to a specific haz3lcore concept